test/automate/automate-utils.js

```diff
@@ -20,6 +20,8 @@
  * no longer need Xvfb in the travis.yml file.
  */
 
+// no idea why this triggers.
+// eslint-disable-next-line import/unambiguous
 let screenShotUniq = 0;
```

Comment:
Thanks @jnmorse . The documentation on unambiguous has broken links, so I tried updating all the eslint dev dependencies, and that resulted in the whole file being reported, not just the variable. I was able to suppress the warning by removing `"sourceType": "module"` from .eslintrc. Do you think it's ok not to have that parserOption ?

Reply:
When I first added the package and tried it locally without the sourceType eslint complains about the import statements. Airbnb config actually has this rule turned off completely.

this is the comment on the rule in their config

```
// Warn if a module could be mistakenly parsed as a script by a consumer
    // leveraging Unambiguous JavaScript Grammar
    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/unambiguous.md
    // this should not be enabled until this proposal has at least been *presented* to TC39.
    // At the moment, it's not a thing.
```

I'm not even clear on what this rule is supposed to be about.

Reply:
Looks like they will pass without it if I update the eslint package to latest. Will re-push in a moment.